catpol/spiders/cdep_career_history.py

```diff
@@ -0,0 +1,44 @@
+import scrapy
+import logging
+
+import sys
+from scrapy.http import Response
+
+import catpol.loaders as loaders
+import catpol.items as items
+import catpol.http as http
+import string
+import catpol.cmdinput as cmdinput
+from typing import Sequence, TypeVar, Coroutine
+
+
+class CdepCareerHistory(scrapy.Spider):
+    name = 'cdep_career_history'
+
+    def __init__(self, after=None, years=None):
+        super().__init__(after=None, years=None)
+        self.years = cmdinput.expand_years(after, years)
+
+    def start_requests(self) -> Coroutine:
+        for page in list(self.revert_pagination(self.years)):
+            yield http.Reqo(url=page, callback=self.get_politician_urls)
+
+    def revert_pagination(self, years: list) -> list:
+        return ['http://www.cdep.ro/pls/parlam/structura2015.de?leg={}&par=A{}'.format(year, char)
+                for year in years
+                for char in string.ascii_uppercase]
+
+    def get_politician_urls(self, response: Response) -> Coroutine:
+        for a in response.css('table a'):
```

Comment:
This selector is too general, causes `get_politician_infos` to respond to URLs which do not represent a person.